translations/de.ts

```diff
@@ -2493,12 +2498,12 @@ Leider führen deine Proxyeinstellungen zu Problemen. Bitte ändere deine Einste
     <message>
         <location filename="../src/widget/widget.cpp" line="1666"/>
         <source>Add new circle...</source>
-        <translation>Neuen Kreis hinzufügen...</translation>
+        <translation>Neuen Kreis hinzufügen ...</translation>
     </message>
     <message>
         <location filename="../src/widget/widget.cpp" line="1692"/>
         <source>By Name</source>
-        <translation>Nach Name</translation>
+        <translation>Nach Namen</translation>
```

Comment:
Einspruch. Jede Anwendung, die ich kenne sortiert "nach Name" (und ich kenne sehr viele Anwendungen :)). Wenn du allein schon deinen Dateimanager (Windows Explorer?) öffnest, siehst du das an der Spalte "Name" bzw. "Dateiname". Genauer gesagt findet eine aufsteigende alphanumerische Sortierung nach der Spalte "Name" statt. Bitte ändere das zurück in "Nach Name". Wo ich es gerade gesehen habe: Du sagst doch genauso auch "Nach Typ" und nicht "Nach Typen".